elements/dash-tab-panel/dash-tab-panel.unittest.js

```diff
@@ -0,0 +1,60 @@
+const expect = chai.expect;
+
+describe('dash-tab-panel', function() {
+  before(dashElements.before());
+  after(dashElements.after());
+  beforeEach(function() {
+    this.container.innerHTML = `
+      <dash-tab-panel>
+        <header aria-role="tab" aria-controls="tab1">Tab 1</header>
+        <section id="tab1" aria-role="tabpanel">Content 1</section>
+        <header aria-role="tab" aria-controls="tab2">Tab 2</header>
+        <section id="tab2" aria-role="tabpanel">Content 2</section>
+        <header aria-role="tab" aria-controls="tab3">Tab 3</header>
+        <section id="tab3" aria-role="tabpanel">Content 3</section>
+      </dash-tab-panel>
+    `;
+    this.tabpanel = this.container.querySelector('dash-tab-panel');
+    this.tabs = Array.from(
+      this.container.querySelectorAll('[aria-role="tab"]')
+    );
+    this.panels = Array.from(
+      this.container.querySelectorAll('[aria-role="tabpanel"]')
+    );
+  });
+
+  it('should only have one visible panel initially',
+    dashElements.elementTest('dash-tab-panel', function() {
```

Comment:
Is it possible to use Mocha's `done` callback for async tests to move `elementTest` into the `beforeEach` callback? Then we'd only need to do it once I think.